beacon-chain/p2p/discovery.go

```diff
@@ -398,6 +508,19 @@ func (s *Service) isPeerAtLimit(inbound bool) bool {
 	return activePeers >= maxPeers || numOfConns >= maxPeers
 }
 
+func (s *Service) isBelowThreshold() bool {
```

Comment:
`isBelowThreshold` does not really indicate to what kind threshold the function refers to.

==> `isBelowPeersThreshold`?

Reply:
Also, I think adding a description of what the function do could help the reader, since its content is not so obvious.